src/SignalR/clients/ts/webpack.config.base.js

```diff
@@ -40,7 +40,10 @@ module.exports = function (modulePath, browserBaseName, options) {
         },
         resolve: {
             extensions: [".ts", ".js"],
-            alias: options.alias,
+            alias: {
+                "./NodeHttpClient": path.resolve(__dirname, "signalr/src/empty.ts"),
```

Comment:
Review: This saves 3kb for the browser .js files
We could just remove this and not save the bytes if we want

Reply:
I think this is the right amount of clever 👍

Of course, I'd like a more first-class way to do this so we wouldn't even have to to waste bytes on a not-implemented NodeHttpClient, but this is certainly better than bundling the real thing in the browser dist.